### ISSUE TITLE
Not able to apply tags to axe

### ISSUE THREAD

**Issue by fede-serena** [sridharbandi/Java-a11y#35]
Every time I try to use any other tag different than the default ones I'm getting an empty report:

`axeRunner.setTags(AxeTag.WCAG21AA);`

**Comment by sridharbandi** [comment]
Can you let me know the url you are using?

**Comment by fede-serena** [comment]
Try with google.

This is what I get if I run with default settings without setting AxeTags
![image](https://github.com/user-attachments/assets/b6727069-dc88-4f1e-bf3b-1971d3cc372b)


And this is what I get if I set 21AA
![image](https://github.com/user-attachments/assets/0b3de223-70a3-43e0-93eb-7b2067c275e7)

**Comment by sridharbandi** [comment]
Set the tag as below. I can see it works.
`axeRunner.setTags(AxeTag.SECTION508);`

**Comment by fede-serena** [comment]
I see it working also if I use SECTION508. But if try with this one: axeRunner.setTags(AxeTag.WCAG21AA); it doesn't return anything

**Comment by sridharbandi** [comment]
> I see it working also if I use SECTION508. But if try with this one: axeRunner.setTags(AxeTag.WCAG21AA); it doesn't return anything

It might be because, it didn't have any violations with that Tag. I tested it with few other URLs and can see setTags is working
![image](https://github.com/user-attachments/assets/92f64f0e-eb8b-4cc6-a190-ace9d3ee19c0)

**Comment by fede-serena** [comment]
I do have violations in the page I'm testing, I know because with the chome extension I do see errors.

Also, you are trying with WCAG2AA, could you please try with WCAG21AA?

**Comment by sridharbandi** [comment]
> I do have violations in the page I'm testing, I know because with the chome extension I do see errors.
> 
> Also, you are trying with WCAG2AA, could you please try with WCAG21AA?

I am not seeing any issues in the chrome plugin.
![image](https://github.com/user-attachments/assets/c779c5ae-22c8-4953-914c-214037ca81d8)

**Comment by fede-serena** [comment]
let's try with this page: https://paraisolinux.com/que-es-una-vpn-y-cual-es-recomendable-para-linux/

with chrome extension I'm getting this:
![image](https://github.com/user-attachments/assets/39f82041-775b-4dc7-a8f8-22bd422d1948)

but with this package I'm getting this:
![image](https://github.com/user-attachments/assets/59613793-da6b-47c8-aef3-36a4cd1a2739)

**Comment by sridharbandi** [comment]
> let's try with this page: https://paraisolinux.com/que-es-una-vpn-y-cual-es-recomendable-para-linux/
> 
> with chrome extension I'm getting this: ![image](https://private-user-images.githubusercontent.com/114818965/375425957-39f82041-775b-4dc7-a8f8-22bd422d1948.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg1NzE2MDEsIm5iZiI6MTcyODU3MTMwMSwicGF0aCI6Ii8xMTQ4MTg5NjUvMzc1NDI1OTU3LTM5ZjgyMDQxLTc3NWItNGRjNy1hOGY4LTIyYmQ0MjJkMTk0OC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxMFQxNDQxNDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZWJjZDgxNDk1ODQ3NDVkODIyNDg0NDljMzc0N2M2MTlhNGJlMzZmNTA3ZGM0ZTQyN2QxZWMxYmJlMzVjZjUyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.A68ART8B8gHqz8tSEUZ6sQ4suclB1u8iZyXI1ADclKM)
> 
> but with this package I'm getting this: ![image](https://private-user-images.githubusercontent.com/114818965/375426238-59613793-da6b-47c8-aef3-36a4cd1a2739.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg1NzE2MDEsIm5iZiI6MTcyODU3MTMwMSwicGF0aCI6Ii8xMTQ4MTg5NjUvMzc1NDI2MjM4LTU5NjEzNzkzLWRhNmItNDdjOC1hZWYzLTM2YTRjZDFhMjczOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxMFQxNDQxNDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iMjU0MTMxZDQxYzA1ZDIzODY5NjA4MTlkMmY3NGNkNzQ4OWE0YzE2MjkxMTExZmI0Zjg0ZmQxNmQzMDAzM2FjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.W5UDDG0cDZvH6rh2gJxqYhg5pHogIIpYceNM_UzXVFk)

Not exactly sure, why the Chrome extension shows WCAG 2.1 AA. But the results are showing for WCAG2AA, WCAG2A and WCAG2AAA.
![image](https://github.com/user-attachments/assets/928ee5d0-b8c4-4f78-937b-55207f290eb5)

**Comment by fede-serena** [comment]
ok, but what happens with the java-a11y package? are you getting any result if you set axeRunner.setTags(AxeTag.WCAG21AA);=?

**Comment by sridharbandi** [comment]
> ok, but what happens with the java-a11y package? are you getting any result if you set axeRunner.setTags(AxeTag.WCAG21AA);=?

No Results. As I quoted earlier, even in the chrome plugin, it was mentioned WCAG 2.1 AA, but the results are not matching with WCAG 2.1 AA. Not sure how that plugin works.
